ec2/sc-ec2-windows-jumpcloud.yaml

```diff
@@ -103,7 +97,7 @@ Resources:
                  [cfn-auto-reloader-hook]
                  triggers=post.update
                  path=Resources.WindowsInstance.Metadata.AWS::CloudFormation::Init
-                 action=cfn-init.exe -v --stack ${AWS::StackId} --resource WindowsInstance --region ${AWS::Region} --configsets SetupCfn,SetupApps,SetEnv,SetupJumpcloud
+                 action=cfn-init.exe -v --stack ${AWS::StackId} --resource WindowsInstance --region ${AWS::Region} --configsets SetupCfn,SetupApps,SetupJumpcloud
```

Comment:
Should the stack flag reference `${AWS::StackName}`, like in the Linux instances above?

Reply:
AWS doc says `cfn-init --stack|-s stack.name.or.id`
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-init.html